example-app/ios/ExampleApp/Info.plist

```diff
@@ -20,6 +20,21 @@
 	<string>1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>http</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>http</string>
+				<string>truidtest</string>
```

Comment:
Do we need to have truidttest anyway? Doesn't it work wit http?

Reply:
I didn't get http to work, never managed to open the app

Reply:
But I can take another stab at it now that i know i missed the native part